src/libraries/System.Security.Cryptography.Csp/src/GeneratePlatformNotSupportedAssemblyHeader.txt

```diff
@@ -0,0 +1,9 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+// ------------------------------------------------------------------------------
+// Changes to this file must follow the https://aka.ms/api-review process.
+// ------------------------------------------------------------------------------
+
+#pragma warning disable 0809
```

Comment:
It'd be cleaner to just set `<NoWarn>$(NoWarn);CS0809</NoWarn>` for this when building PNSE.